tutorial/cpp/6-HNSW.cpp

```diff
@@ -55,12 +55,10 @@ int main() {
             printf("\n");
         }
 
-        index.search(nq, xq, k, D, I);
-
-        printf("I=\n");
+        printf("D=\n");
         for (int i = nq - 5; i < nq; i++) {
             for (int j = 0; j < k; j++)
-                printf("%5zd ", I[i * k + j]);
+                printf("%5zd ", D[i * k + j]);
```

Comment:
Format specifies type 'ssize_t' (aka 'long') but the argument has type 'float' [-Werror,-Wformat]
                printf("%5zd ", D[i * k + j]);
                        ~~~~    ^~~~~~~~~~~~
                        %5f

Reply:
@saarthdeshpande would you like to fix this part?

Reply:
Sure

Reply:
Done! Added the changes for other tutorial files too.